### PR TITLE
#2371 Issue : Migrated admin jobs table to mat-table

### PR DESCRIPTION
**Pull request by Basimohd**
I've migrated Jobs Table of Admin Control to mat-table. Could you kindly review and confirm if this approach aligns with the desired implementation?

### CODE REVIEW

File: apps/client/src/app/components/admin-jobs/admin-jobs.html [hunk 1]
```diff
@@ -13,122 +13,163 @@
...
-            <th class="mat-header-cell px-1 py-2" i18n>Data Source</th>
-            <th class="mat-header-cell px-1 py-2 text-right" i18n>Attempts</th>
-            <th class="mat-header-cell px-1 py-2" i18n>Created</th>
-            <th class="mat-header-cell px-1 py-2" i18n>Finished</th>
-            <th class="mat-header-cell px-1 py-2" i18n>Status</th>
-            <th class="mat-header-cell px-1 py-2">
+      <table class="gf-table w-100" mat-table [dataSource]="dataSource">
+        <ng-container matColumnDef="index">
+          <th *matHeaderCellDef class="px-1 py-2 text-right" mat-header-cell>
+            <ng-container>#</ng-container>
```
Comment (dtslvr) [inline apps/client/src/app/components/admin-jobs/admin-jobs.html:19]: When there is no `i18n` directive, you don't need `<ng-container>`. Can you remove it?

**Review by dtslvr — CHANGES_REQUESTED** [review]
That looks fine, @Basimohd! 👍🏻 

Just one comment and a little problem with the formatting in `admin-jobs.module.ts` (space in `{ }`) 🙂

**Review by dtslvr — APPROVED** [review]

**Comment by Basimohd** [comment]
@dtslvr should I do the formatting?

**Comment by Basimohd** [comment]
@dtslvr I have noticed that one checks failed on formatting.Should I have to fix it?

**Pull request merged**